src/index.html

```diff
@@ -11,7 +11,166 @@
+            >
+          </label>
+        </div>
+        <div class="input-field">
+          <label>
+            How old are You?
+            <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+            >
+          </label>
+        </div>
+        <div class="input-field">
+          <label>
+            Full date of birth:
+            <input type="date" name="birth">
+          </label>
+        </div>
+        <div class="input-field">
+          <label>
+          I accept the term of the agreement
+            <input type="checkbox" name="agreement">
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="form-group">
+        <legend>
+        Registration
+        </legend>
+        <div>
+          <label >
+            E-mail:
+            <input
+            type="email"
+            name="useremail"
+            placeholder="email@example.com"
+            >
+          </label>
+        </div>
+        <div class="input-field">
+          <label>
+            Password:
+            <input type="password" name="userpassword">
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="form-group">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <div>
+          <label>
+            Do you love cats?
+            <input type="radio" name="answer">
+            <label>Yes</label>
+            <input type="radio" name="answer">
+            <label>No</label>
+          </label>
+        </div>
+        <div class="input-field">
+          <label>
+            What is your favorite color?
+            <input type="color" name="favcol">
+          </label>
+        </div>
+        <div class="input-field">
+          <label>
+            What time do you go to bed?
+            <input
+            type="time"
+            name="bedtime"
+            step="1"
+            >
+          </label>
+        </div>
+        <div class="input-field">
+          <label>
+            What are your favorite brand  of cars?
+            <select name="car" multiple>
+              <option value="bmv">
+                BMW
+              </option>
+              <option value="audi">
+                Audi
+              </option>
+              <option value="lada">
+                Lada
+              </option>
+            </select>
+          </label>
+        </div>
+        <div class="input-field">
+          <label>
+            How do you rate our work?
+            <input type="range" name="rateofwork">
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="form-group">
+        <legend>
+          Additional info:
+        </legend>
+        <div>
+          <label class="input">
+            Comments:
+            <textarea
+            class="commentsarea"
+            minlength="50"
+            maxlength="1000"
+            >
+            </textarea>
```

Comment:
now this textaraea has redundant space
<img width="285" alt="image" src="https://user-images.githubusercontent.com/102411130/223352754-c2d61c41-6489-4557-a6f8-98fa29554c4a.png">